content/reference/v1beta1/helmchart.md

```diff
@@ -25,6 +25,9 @@ spec:
 
   exclude: "repl{{ ConfigOptionEquals `include_chart` `include_chart_no`}}"
 
+  # helmVersion identifies the Helm Version used to render the Chart. Default is v2.
+  helmVersion: v3
```

Comment:
Since v3 is beta, I'm thinking it's best to show this `helmVersion` field, but use `v2` in the example.

Reply:
but that would be default... why not just omit it then?

Reply:
By including "helmVersion: v2", you're documenting how the setting works. If you omit it, it's less explicit (and therefore less useful).